docs/product/releases/index.mdx

```diff
@@ -66,7 +66,7 @@ The "Resolved in the current release" option records the current release that's
 
 ### Resolved in the Next Release
 
-The "Resolved in the next release" option works in a similar way to resolved in the current release. But because we can't record the resolution version since it hasn't been created yet, we record the current release version and update it when a newer release is created.
+The "Resolved in the next release" option works in a similar way to resolved in the current release. But because we can't record the resolution version since it hasn't been created yet, we record the latest release version the group was seen in and update it when a newer release is created.
```

Comment:
```suggestion
The "Resolved in the next release" option works similarly to resolving in the current release. But because we can't record the resolution version since it hasn't been created yet, we record the latest release version the group was seen in and update it when a newer release is created. When using semantic versioned releases, the group will be resolved with the next release which is greater than all existing ones.
```

For semver releases, it is not considered the last release the group was seen but just the greatest semver release to be released [see test](https://github.com/getsentry/sentry/blob/7535e495e365c2fc7046a951df90d8eccdb5be65/tests/sentry/issues/endpoints/test_group_details.py#L373).

For semver releases I would add this:
> When using semantic versioned releases, the group will be resolved with the next release which is greater than all existing ones.